source/_components/camera.environment_canada.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Environment Canada Radar"
+description: "Radar imagery from Environment Canada."
+date: 2017-07-07 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: environment_canada.jpg
+ha_category: Weather
+ha_release: 0.74
+ha_iot_class: "Cloud Polling"
+---
+
+The `environment_canada` camera platform uses [Environment Canada](https://weather.gc.ca/mainmenu/weather_menu_e.html) as a source for current meteorological radar imagery.
+
+- The platform automatically determines which radar station to use, according to this hierarchy:
+  - Station code from sensor configuration
+  - Closest station to latitude / longitude from sensor configuration
+  - Closest station to latitude / longitude from Home Assistant core configuration
+
+- If no name is given, the camera entity will be named `camera.<station_name>_radar`.
+
+To add Environment Canada radar imagery to your installation, add the desired lines from the following example to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  - platform: environment_canada
+    latitude: 50
+    longitude: -100
+    station: XFT
+    name: Ottawa Radar
+    loop: True
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration